cEP-0024.md

```diff
@@ -0,0 +1,82 @@
+#Implement GitMate automations and plugins for coala
+
+| Metadata |                                                       |
+| -------- | ----------------------------------------------------- |
+| cEP      | 24                                                    |
+| Version  | 1.0                                                   |
+| Title    | Implement Gitmate automations and plugins for coala   |
+| Authors  | Vamshi Krishna Bommerla <mailto:bommvams72@gmail.com> |
+| Status   | Proposed                                              |
+| Type     | Process                                               |
+
+##Abstract
+
+This cEP describes the details of the process of creating new automations and
+plugins in GitMate which helps coala development workflow, as a part of the
+[GSoC project](https://summerofcode.withgoogle.com/projects/#5344316227059712).
+
+## Introduction
+
+GitMate.io is an automation tool for developers. While it works well, there are
+a number of plugins which saves a lot of time of our
+community(for example automatically close PRs from master). Many of them are
+general plugins, which are required by every org, but we need more plugins
+which support coala development workflow.
+
+## New Features
+
+### Auto-assign issue based on commit mention
+
+When a commit mentions(fixes,closes) an issue, auto assign unassigned issues.
```

Comment:
will it provide a comment on the issue that coala can configure to suit our needs?

and what happens if they are not an org member, and cant be assigned?

a comment pinging maintainers?

a gitmate managed queue for maintainers to check?

Reply:
@jayvdb 
Yes, a configurable comment on the issue when user is assigned.

If they are not in the org, they will be informed to get invited.

I am thinking of a directly assigning the user to the issue.